lib/kitchen/example_element.rb

```diff
@@ -27,9 +27,15 @@ def self.short_type
     #
     # @return [ElementEnumerator]
     #
-    def titles
-      search("[data-type='title']")
```

Comment:
this can be removed because it's the same implementation as `ElementBase#titles`